core/src/main/java/com/novoda/merlin/RxCallbacksManager.java

```diff
@@ -0,0 +1,25 @@
+package com.novoda.merlin;
+
+import rx.Observable;
+import rx.subjects.BehaviorSubject;
+
+public class RxCallbacksManager {
+
+    private final BehaviorSubject<Boolean> merlinRxConnectionStatusSubject;
```

Comment:
I can see how that would be useful. But I think true/false is enough for the simple use case of conveying connection status. I think we should defer this and add it in if/when transitional states are supported.

But it's your call :-)